specification/resourcegraph/resource-manager/readme.md

```diff
@@ -50,13 +49,15 @@ These settings apply only when `--tag=package-2019-04` is specified on the comma
 input-file:
   - Microsoft.ResourceGraph/stable/2019-04-01/resourcegraph.json
 ```
+
 ### Tag: package-2018-09-preview
 
 These settings apply only when `--tag=package-2018-09-preview` is specified on the command line.
 
 ``` yaml $(tag) == 'package-2018-09-preview'
 input-file:
-- Microsoft.ResourceGraph/preview/2018-09-01-preview/resourcegraph.json
+  - Microsoft.ResourceGraph/preview/2018-09-01-preview/resourcegraph.json
+  - Microsoft.ResourceGraph/preview/2018-09-01-preview/savedquery.json
```

Comment:
Since we have a separate swagger for the queries; How will it be included in the SDK generations?

Reply:
@chiragg4u Do you have any suggestion on this?